src/FilterService.Broker/Requests/UserService.cs

```diff
@@ -21,6 +21,30 @@ public class UserService : IUserService
     private readonly IGlobalCacheRepository _globalCache;
     private readonly IRequestClient<IFilteredUsersDataRequest> _rcGetUsers;
 
+    private string CreateRedisKey(
+      List<Guid> usersIds,
+      int skipCount,
+      int takeCount,
+      bool? ascendingSort,
+      string fullNameIncludeSubstring)
+    {
+      List<object> additionalArgs = new() { skipCount, takeCount };
+
+      if (ascendingSort.HasValue)
+      {
+        additionalArgs.Add(ascendingSort.Value);
+      }
+
+      if (!string.IsNullOrEmpty(fullNameIncludeSubstring))
+      {
+        additionalArgs.Add(fullNameIncludeSubstring);
+      }
+
+      var key = usersIds.GetRedisCacheHashCode(additionalArgs.ToArray());

```

Comment:
May be remove this varible and "return usersIds.GetRedisCacheHashCode(additionalArgs.ToArray());"?